### PR TITLE
Qa widget

### PR DESCRIPTION
**Pull request by mattmluu**
A lot of refactoring
few changes to styled components
add question modal

### CODE REVIEW

File: src/components/QA/QA.jsx [hunk 1]
```diff
@@ -77,23 +95,99 @@ class QA extends Component {
     return questions;
   }
```
Comment (charliebailey24) [inline src/components/QA/QA.jsx:97]: Suggestion: I recently learned that you can actually add a name to your events and then use the ES6 computed property technique to set your state variables to the event.target.value (so long as your event names are the same as your state variable names), all with a single event handler. This saves you from having to write out so many dang event handlers. See the attached screenshot for an example.
Screen Shot 2022-04-09 at 4 01 04 PM

**Review by charliebailey24 — APPROVED** [review]
Just added one suggestion that can save you from having to write out so many event handlers.

File: src/components/QA/QA.jsx (hunk 1 above)
Comment (mattmluu) [reply src/components/QA/QA.jsx:97]: ooooo that is so much cleaner. I'll try to refactor when I finish everything else

Comment (charliebailey24) [reply src/components/QA/QA.jsx:97]: hahah no need to make more work right now. I just thought I'd let you know if you need to add more event handlers in the future!

**Pull request merged**